theano/gpuarray/opt.py

```diff
@@ -344,6 +380,9 @@ def apply(self, fgraph):
             for new_o, old_o in zip(outputs, node.outputs):
```

Comment:
so here to know the number of new nodes the optimization created, you can do something like:

if new_ops: # if we moved it to the GPU
   node_created[lopt] += len(theano.gof.ops([mapping[i] for i in node.inputs], outputs)) # theano.gof.ops return a set a Apply nodes)